src/components/Counter.js

```diff
@@ -20,10 +20,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment(this.props.count) }>
```

Comment:
All you need to do here is invoke the increment and decrement methods. These methods do not need parameters, all they do is trigger an action, which is sent to the reducer, which handles the updates to the store. `<button onClick={() => this.props.increment() }>` is all that is necessary here.